templates/e-commerce/data.json

```diff
@@ -1,13 +1,17 @@
 {
-  "home": {
+  "header": {
     "head-tag": {
-      "title": "branding",
+      "title": "Commerce",
+      "canonical-path": "templates/themes/3/home.amp",
+      "extensions": [
+        "amp-sidebar"
+      ],
       "font-stylesheets": [
         "https://fonts.googleapis.com/css?family=Open+Sans+Condensed:300,700"
-      ]
+      ],
+      "css-path": "templates/e-commerce/page.css"
     },
     "navbar": {
-      "svg-trigger": "<svg xmlns='http://www.w3.org/2000/svg' width='24' height='24' viewBox='0 0 24 24'><path fill='none' d='M0 0h24v24H0z'/><path fill='currentColor' d='M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z'/></svg>",
```

Comment:
Removed this since the alignment in our templates wasn't correct. Need to find the correct styles in original PR and add.